drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5871,7 +5871,7 @@ void RasterizerStorageGLES2::initialize() {
 	config.float_texture_supported = config.extensions.has("GL_ARB_texture_float") || config.extensions.has("GL_OES_texture_float");
 	config.s3tc_supported = config.extensions.has("GL_EXT_texture_compression_s3tc") || config.extensions.has("WEBGL_compressed_texture_s3tc");
 	config.etc1_supported = config.extensions.has("GL_OES_compressed_ETC1_RGB8_texture") || config.extensions.has("WEBGL_compressed_texture_etc1");
-	config.pvrtc_supported = config.extensions.has("IMG_texture_compression_pvrtc") || config.extensions.has("WEBGL_compressed_texture_pvrtc");
+	config.pvrtc_supported = config.extensions.has("GL_IMG_texture_compression_pvrtc") || config.extensions.has("IMG_texture_compression_pvrtc") || config.extensions.has("WEBGL_compressed_texture_pvrtc");
```

Comment:
I think the one without `GL_` prefix might have been a bug, so it could likely be removed. All OpenGL extensions start with `GL_` IINM.

That's also what is used in the GLES3 backend:
```
config.pvrtc_supported = config.extensions.has("GL_IMG_texture_compression_pvrtc");
```
(BTW the `WEBGL_*` check should maybe also be added for GLES3?)